### PR TITLE
Add weekStartsOn prop

### PR DESCRIPTION
**Pull request by thecoorum**
## Description
This PR adds `weekStartsOn` prop to define what day index the week should start

### CODE REVIEW

File: src/Month.tsx [hunk 1]
```diff
@@ -27,10 +32,15 @@ export const MonthPage = React.memo(({ index }: { index: number }) => {
...
+      eachWeekOfInterval(
+        {
+          start: firstDayOfMonth,
+          end: lastDayOfMo,
+        },
+        {
+          weekStartsOn,
+        }
+      ),
     [firstDayOfMonth, lastDayOfMo]
```
Comment (computerjazz) [inline src/Month.tsx:44]: please add `weekStartsOn` to dependency array

**Review by computerjazz — CHANGES_REQUESTED** [review]
looks great, thanks! just one small change

**Comment by computerjazz** [comment]
Thank you! I'm assuming if `weekStartsOn` is undefined the default (Sunday/0) will apply?

**Comment by thecoorum** [comment]
> Thank you! I'm assuming if `weekStartsOn` is undefined the default (Sunday/0) will apply?

Yes, this is what official docs of date-fns says

**Pull request merged**